File: PArticulo/Articulo.cs

using System;

namespace PArticulo{

	public class Articulo{

		public Articulo (long id, string nombre, decimal? precio, long? categoria){
			this.Id = id;
			this.Nombre = nombre;
			this.Precio = precio;
			this.Categoria = categoria;
		}
		public long Id {get;set;}
		public string Nombre { get; set;}
		public decimal? Precio{ get; set;}
		public long? Categoria{ get; set;}
	}

}

File: HPedido/src/main/java/org/institutoserpis/ad/Articulo.java
package org.institutoserpis.ad;

import javax.persistence.GeneratedValue;
import javax.persistence.Entity;
import javax.persistence.GenerationType;
import javax.persistence.Id;



@Entity
public class Articulo {
	private long id;
	private String nombre;
	private double precio;
	private long categoria;
	
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	
	public long getId() {return id;}
	public String getNombre() {return nombre;}
	public double getPrecio() {return precio;}
	public long getCategoria() {return categoria;}
	
	public void setId(long id) { this.id=id;}
	public void setNombre(String nombre) {this.nombre=nombre;}
	public void setPrecio(double precio) {this.precio=precio;}
	public void setCategoria(long categoria) {this.categoria=categoria;}
	
}
